garak/probes/visual_jailbreak.py

```diff
@@ -65,7 +65,7 @@ def _load_SafeBench(self):
         ):
             filepath = safebench_data_dir / filename
             if not os.path.isfile(filepath):
-                uri = f"https://raw.githubusercontent.com/ThuCCSLab/FigStep/main/data/images/SafeBench/{filename}"
+                doc_uri = f"https://raw.githubusercontent.com/ThuCCSLab/FigStep/main/data/images/SafeBench/{filename}"
                 with open(filepath, "wb") as f:
                     f.write(requests.get(uri).content)
```

Comment:
```suggestion
                    f.write(requests.get(data_uri).content)
```

Reply:
We overloaded `doc_uri` here with the path to the data (instead of the reference). This is what's causing the failing tests.